src/error.rs

```diff
@@ -59,6 +61,12 @@ impl From<a2::Error> for Error {
     }
 }
 
+impl From<fcm::FcmError> for Error {
```

Comment:
I'd suggest using [`thiserror`](https://crates.io/crates/thiserror) for `Error` and its `#[from]` macro for automatic `From` implementations.